### PR TITLE
Added eligibility exit points

### PR DESCRIPTION
**Pull request by tony-gledhill**

### CODE REVIEW

File: pages/apply/[resident]/summary.tsx [hunk 1]
```diff
@@ -27,6 +32,36 @@ const UserSummary = (): JSX.Element => {
...
+    [applicants]
+  );
+
+  const mainApplicant = applicants
+    .filter((app) => app === application.mainApplicant)
+    .map((ma) => ma)
+    .reduce((map) => map);
+
+  const onConfirmData = () => {
+    const isMainApplicantEligible = eligibilityMap.get(mainApplicant);
```
Comment (tcmorris) [inline pages/apply/[resident]/summary.tsx:57]: I think we need to check if all members involved in the application are eligible, not just the main applicant.

File: pages/apply/[resident]/summary.tsx [hunk 2]
```diff
@@ -62,6 +97,7 @@ const UserSummary = (): JSX.Element => {
       <MedicalNeedsSummary currentResident={currentResident} />
 
       <ButtonLink href="/apply/overview">I confirm this is correct</ButtonLink>
+      <Button onClick={onConfirmData}>I confirm this is correct Button</Button>
```
Comment (tcmorris) [inline pages/apply/[resident]/summary.tsx:100]: To be updated? duplicate buttons

Comment (tony-gledhill) [reply pages/apply/[resident]/summary.tsx:100]: My bad. This has been removed

Comment (tcmorris) [reply pages/apply/[resident]/summary.tsx:96]: sorry, was referring to what looks like `ButtonLink` that needs removing and `DeleteLink` reinstated to match designs.

**Pull request merged**